digitaltwin_client/src/dt_interface_client.c

```diff
@@ -1188,6 +1189,47 @@ static void ProcessPropertyUpdateIfNeededFromDesired(DT_INTERFACE_CLIENT* dtInte
     }
 }
 
+// For each high-level property associated with this interface, determine if there is a callback function associated with it.
+static void ProcessPropertyUpdateIfNeededFromReportedOnly(DT_INTERFACE_CLIENT* dtInterfaceClient, JSON_Object* dtInterfaceReportedJson, size_t propertyNumber, int jsonVersion)
+{
+    const char* propertyName;
+    if ((propertyName = json_object_get_name(dtInterfaceReportedJson, propertyNumber)) == NULL)
```

Comment:
>json_object_get_name [](start = 24, length = 20)

If we remove the reported value from a previous call to desired/reported, I believe this will cause a LogError() to be output correct?  We shouldn't do that as it'd be a false positive for customer.

Reply:
No.  Refer to line 1280, after ProcessPropertyUpdateIfNeededFromDesired(), we do a json_object_get_count() to get the number of reported property in the json before going into another for loop for processing the reported-only properties.  The index is restarted.  The LogError should not be false positive in this case.